code/script.js

```diff
@@ -218,77 +229,178 @@ const generateBoard = () => {
           <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  secret =
+    charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
+
+function setTime() {
+  ++totalSeconds;
+  secondsLabel.innerHTML = pad(totalSeconds % 60);
+  minutesLabel.innerHTML = pad(parseInt(totalSeconds / 60));
+}
+
+function pad(time) {
+  let timeString = time + "";
+  if (timeString.length < 2) {
+    return "0" + timeString;
+  } else {
+    return timeString;
+  }
+}
+
+function findOutCounter() {
+  updateDisplay(++counterValue);
+}
+
+function updateDisplay(val) {
+  document.getElementById("counterLabel").innerHTML = val;
```

Comment:
A pretty advanced way of doing timer, but great that you implemented it and understood!